index.css

```diff
@@ -222,3 +220,745 @@ section img {
   top: 4px;
   left: 0;
 }
+
+.recent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0;
+  gap: 80px;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  left: 0;
+  top: 682px;
+}
```

Comment:
- [ ] It looks like the contents of your works section is aligned to the left. kindly align them to the center to match the design template. You are expected to stick to figma design as much as possible. Check the images below.
- [ ] Kindly Fix your button size and also the list of technologies in the project section in the image below to match the design template, please.
### Your design
![2](https://user-images.githubusercontent.com/94551484/192063738-ad4fa132-05bc-4dd0-aa9a-76e7bf67cf05.png)
### Figma template
![f2](https://user-images.githubusercontent.com/94551484/192063757-7d57ea35-1fc9-4a32-9b44-199fb18e3e95.png)


- [ ] Please use grid to lay out the contents of this section as demanded by the project requirements. You can use `display: grid;`